src/component/base/DraftEditor.react.js

```diff
@@ -202,14 +202,9 @@ class DraftEditor extends React.Component<DraftEditorProps, State> {
   _renderPlaceholder(): ?React.Element<any> {
     if (this._showPlaceholder()) {
       return (
-        /* $FlowFixMe(>=0.53.0 site=www,mobile) This comment suppresses an
-         * error when upgrading Flow's support for React. Common errors found
-         * when upgrading Flow's React support are documented at
-         * https://fburl.com/eq7bs81w */
         <DraftEditorPlaceholder
           text={nullthrows(this.props.placeholder)}
           editorState={this.props.editorState}
-          textAlignment={this.props.textAlignment}
```

Comment:
I think we still want to do this with `textAlignment` - and I did see uses of `textAlignment` internally at FB.

Reply:
The reason why I remove was due to the fact that `DraftEditorPlaceholder` does not use the prop `textAlignment`.

If you refer to `https://github.com/facebook/draft-js/blob/master/src/component/base/DraftEditorPlaceholder.react.js` you will see no usage of `textAlignment` apart from the props definition itself

But if you prefer I can updated this PR to still have to pass `textAlignment` and make it a nullable `?textAlignment` instead.